style.css

```diff
@@ -390,25 +355,116 @@ hr {
   padding: 8px;
 }
 
+#submit {
+  width: 110px;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+}
+
+/*  Navbar Hamburger Css ::clicked */
+.click {
+  background: rgba(96, 112, 255, 0.8);
```

Comment:
- [ ] Great job so far. kindly use the same figma background color (#6070FF) so it follows the design